lib/index.js

```diff
@@ -32,7 +34,13 @@ function main({ port, client } = {}) {
   let filters = {};
   if (config.accept) {
     debug('loading rules from %s', config.accept);
-    filters = require(path.resolve(process.cwd(), config.accept));
+    const acceptLocation = path.resolve(process.cwd(), config.accept);
+
+    if (acceptLocation.endsWith(".yaml") || acceptLocation.endsWith(".yml")) {
+        filters = yaml.safeLoad(fs.readFileSync(acceptLocation, 'utf8'))
+    } else {
+        filters = require(acceptLocation);
+    }
```

Comment:
YAML is a superset of JSON (i.e. valid JSON is valid YAML), so we don't need to check file extensions – reading the file with `yaml.safeLoad` will work for both JSON and YAML.